_guides/https.md

```diff
@@ -7,24 +7,33 @@ description: Easily configure The Lounge to be served over HTTPS for better secu
 In this guide, we will see how to easily configure The Lounge to be served over [HTTPS](https://en.wikipedia.org/wiki/HTTPS) for better security and privacy.
 
 {: .alert.alert-warning role="alert"}
-The Lounge only has basic HTTPS support, and will need to be manually restarted to reload certificates on renewal. For advanced HTTPS support, consider [using a reverse proxy](/docs/guides/reverse-proxies).
+The Lounge only has basic HTTPS support.
 
```

Comment:
Please don't remove the reverse proxy docs, that's actually what should be referred to, not just an nginx config that only works on some distributions.